test/partial-affinity.js

```diff
@@ -103,6 +103,26 @@ allocCluster.test('nominal parameters', {
     }
 });
 
+var hundredAlice = [];
+for (var _count = 0; _count < 30; _count++) {
+    hundredAlice.push('alice');
+}
+
+allocCluster.test('big cluster', {
+    size: 30,
+    kValue: 15,
+    namedRemotes: hundredAlice,
```

Comment:
manyAlices. Specifically 30, but that’s not important right now. https://www.youtube.com/watch?v=VOmD-xqK2Es